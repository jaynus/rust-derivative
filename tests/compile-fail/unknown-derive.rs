#[cfg(feature = "use_core")]
extern crate core;

#[macro_use]
extern crate derivative;

#[derive(Derivative)]
#[derivative(DoesNotExist1)]
struct Foo;

#[derive(Derivative)]
#[derivative(DoesNotExist2(with_some="argument"))]
struct Bar;

fn main() {}