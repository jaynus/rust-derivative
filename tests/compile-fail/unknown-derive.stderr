error: Deriving `DoesNotExist1` is not supported by derivative
 --> $DIR/unknown-derive.rs:8:14
  |
8 | #[derivative(DoesNotExist1)]
  |              ^^^^^^^^^^^^^

error: Deriving `DoesNotExist2` is not supported by derivative
  --> $DIR/unknown-derive.rs:12:14
   |
12 | #[derivative(DoesNotExist2(with_some="argument"))]
   |              ^^^^^^^^^^^^^

error: Deriving `DoesNotExist3` is not supported by derivative
  --> $DIR/unknown-derive.rs:18:18
   |
18 |     #[derivative(DoesNotExist3)]
   |                  ^^^^^^^^^^^^^
